### PR TITLE
Bug 1381314- Link "Permissions" from user-profile to get_permissions.html

### PR DESCRIPTION
**Pull request by Dipanks**

### CODE REVIEW

**Comment by Dipanks** [comment]
How to clear these dylan? rebase is not working probably.. Shall I open a new pull request?

**Comment by emceeaich** [comment]
The pull request now only has one file and change in it. I'll add the PR so it works in splinter.

**Comment by Dipanks** [comment]
@emceeaich, do I need to change anything?

File: extensions/UserProfile/template/en/default/pages/user_profile.html.tmpl [hunk 1]
```diff
@@ -108,6 +108,13 @@
       [% END %]
     </td>
   </tr>
+[% ELSE %]
+  <tr>
+    <th>Get Permissions</th>
+    <td>
```
Comment (dylanwh) [inline extensions/UserProfile/template/en/default/pages/user_profile.html.tmpl]: pretty sure there is a colspan="2" needed here, were you able to test this? need help with the vagrant vm?

Comment (Dipanks) [inline extensions/UserProfile/template/en/default/pages/user_profile.html.tmpl]: I think I tested it before creating the pr. But I will test it once more and let you know in irc

**Comment by Dipanks** [comment]
Is this okay? 
Or the 'Merge branch 'master'.....' will be a problem?
Do I have to rebase, in case if it is a problem?

**Pull request closed without merging**